components/Player/PlayerActions.jsx

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
```

Comment:
Not a huge fan of this being called PlayerActions.jsx since it may confuse people into thinking this is somehow related to redux actions. Not a big deal but something to think about.